scripts/insert_into_appointments.sql

```diff
@@ -1 +1,17 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(start_at, length, patient_id, doctor_id)
```

Comment:
`patient_id` and `doctor_id` don't exist in your `appointments` table so this will cause an error.